### PR TITLE
Jayden practice mode askmode

### PR DESCRIPTION
**Pull request by JeydinNewWon**
Did the basic question and answer functionality!

### CODE REVIEW

File: src/components/ExitButton.tsx [hunk 1]
```diff
@@ -4,6 +4,6 @@ import { IoCaretBack } from 'react-icons/io5';
 export function ExitPage() {
     const navigate = useNavigate();
     return (
-        <button onClick={() => navigate('/')} className="top-left" style={{position:"absolute", top:"5%", left:"2.5%", fontSize:"30px"}}> <IoCaretBack style={{display: "inline-block"}}/>Exit</button>
+        <button onClick={() => { localStorage.clear(); navigate('/');}} className="top-left" style={{position:"absolute", top:"5%", left:"2.5%", fontSize:"30px"}}> <IoCaretBack style={{display: "inline-block"}}/>Exit</button>
```
Comment (TAS-scorchedshadow) [inline src/components/ExitButton.tsx:7]: won't this cause issues when tracking the stats? Probably just clear the qData field instead

File: src/components/ExampleComponent.tsx [hunk 2]
```diff
@@ -31,10 +31,10 @@ function ExampleComponent() {
                 question: docData.question ?? "no question data",
                 explanation: docData.explanation ?? "no explanation data",
             }
-            console.log(questionInfo);
             questionData.push(questionInfo);
         });
         setData(questionData);
+        console.log(data);
```
Comment (TAS-scorchedshadow) [inline src/components/ExampleComponent.tsx:37]: Nitpick remove

**Review by TAS-scorchedshadow — CHANGES_REQUESTED** [review]
Just a few things otherwise LGTM

File: src/components/ExitButton.tsx (hunk 1 above)
Comment (JeydinNewWon) [reply src/components/ExitButton.tsx:7]: Oh yeah it will, i'll go and edit it

**Review by TAS-scorchedshadow — APPROVED** [review]
LGTM!

**Pull request merged**